src/components/ProductCard.tsx

```diff
@@ -0,0 +1,41 @@
+import { StorageKeys, setStorageData } from "../../utils/storage";
+import React, { useState } from "react";
+import { View, Text, Image, TouchableOpacity } from "react-native";
+import Counter from "./Counter";
+import { addToCart, removeFromCart } from "../../utils/manageCart";
+import { ProductCardProps } from "../../interfaces/products";
```

Comment:
eu faria setup de absolute imports no projeto

Reply:
absolute imports? podes dar um exemplo?

Reply:
https://dev.to/abrahamlawson/using-absolute-paths-in-react-native-24ak